### PR TITLE
[BE] Task #207446 Ip reassign facilitator to camp with reassign beneficiary changes

### PR DESCRIPTION
**Pull request by GitVivekHub**
# I have ensured that following `Pull Request Checklist` is taken care of before sending this PR

* [x] Code is formatted as per format decided
* [x] Updated acceptance criteria in tracker
* [x] Updated test cases in test-cases-tracker
* [x] Updated new API endpoint if any in common postman collection
* [x] Updated DB changes in db-tracker if any

### CODE REVIEW

File: src/src/camp/camp.core.service.ts [hunk 1]
```diff
@@ -289,4 +289,26 @@ export class CampCoreService {
 
 		return response;
 	}
+
+	public async updateCampStatus(
```
Comment (sagarkoshti1990) [inline src/src/camp/camp.core.service.ts:293]: change `updateCampStatus` to `updateCampGroup`

File: src/src/camp/camp.service.ts [hunk 2]
```diff
@@ -2244,6 +2266,22 @@ export class CampService {
 			if (!updatedResult.success)
 				result.data.unsuccessfulReassignmentIds.push(body?.learner_id);
 
+			if (
+				old_camp_details_repsonse?.data?.group_users?.[0].group
+					?.group_users_aggregate?.aggregate?.count == 0
+			) {
+				await this.campcoreservice.updateCampStatus(
```
Comment (sagarkoshti1990) [inline src/src/camp/camp.service.ts:2273]: change `updateCampStatus` to `updateCampGroup`

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: src/src/camp/camp.service.ts (hunk 2 above)
Comment (GitVivekHub) [reply src/src/camp/camp.service.ts]: resolved

File: src/src/camp/camp.core.service.ts (hunk 1 above)
Comment (GitVivekHub) [reply src/src/camp/camp.core.service.ts]: resolved

File: src/src/camp/camp.service.ts (hunk 2 above)
Comment (GitVivekHub) [reply src/src/camp/camp.service.ts]: resolved

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**